storage/inmem/inmem.go

```diff
@@ -150,6 +150,7 @@ func (db *store) Truncate(ctx context.Context, txn storage.Transaction, params s
 		return err
 	}
 
+	// For backwards compatibility, check if `RootOverwrite` was configured.
```

Comment:
https://github.com/open-policy-agent/opa/pull/5005/files#diff-1bb408d65c986dade9935226d79f06e2a4a5bca16409e2553efb789daccc039bR281 There we have an extra condition. It's not needed here?

Reply:
We should not as the iteration we do next should do the same job as the `overwriteRoot(params.BasePaths)` condition in the disk code.

Reply:
Thanks!